generated/models/SwapResponseDynamicSlippageReport.ts

```diff
@@ -43,8 +43,33 @@ export interface SwapResponseDynamicSlippageReport {
      * @memberof SwapResponseDynamicSlippageReport
      */
     slippageBps?: number;
+    /**
+     * 
+     * @type {string}
+     * @memberof SwapResponseDynamicSlippageReport
+     */
+    categoryName?: SwapResponseDynamicSlippageReportCategoryNameEnum;
+    /**
+     * 
+     * @type {number}
+     * @memberof SwapResponseDynamicSlippageReport
+     */
+    heuristicMaxSlippageBps?: number;
 }
 
+
+/**
+ * @export
+ */
+export const SwapResponseDynamicSlippageReportCategoryNameEnum = {
+    Stable: 'stable',
+    Lst: 'lst',
+    Bluechip: 'bluechip',
+    Verified: 'verified'
+} as const;
+export type SwapResponseDynamicSlippageReportCategoryNameEnum = typeof SwapResponseDynamicSlippageReportCategoryNameEnum[keyof typeof SwapResponseDynamicSlippageReportCategoryNameEnum];
```

Comment:
Remove this, we don't need enum here as it's readable value here, and also this not the best way to construct an enum. We can just do

```
enum SwapResponseDynamicSlippageReportCategoryName {
    STABLE: 'stable',
    LST: 'lst',
    BLUECHIP: 'bluechip',
    VERIFIED: 'verified'
}
```

Reply:
This is autogen though, I only edited the swagger yaml file

Reply:
owh lol, the generated enum looks bad

Reply:
paiseh approved

Reply:
> owh lol, the generated enum looks bad

Yeah haha agreed